src/compiler/parser.ts

```diff
@@ -436,6 +436,13 @@ module ts {
         // attached to the EOF token.
         let parseErrorBeforeNextFinishedNode: boolean = false;
 
+        export const enum StatementFlags {

```

Comment:
True, but that aspect could be achieved by checking that the result of `getDeclarationFlags` is not `StatementFlags.None`. It doesn't have anything to do with the flags that are passed in.